package.json

```diff
@@ -72,7 +72,7 @@
     "@angular/router": "^7.2.13",
     "@hmcts/ccpay-web-component": "5.0.4",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "4.13.8",
+    "@hmcts/ccd-case-ui-toolkit": "4.10.0-rc.6",
```

Comment:
could this be a live version instead of rc?

Reply:
its live yes but i will make a live tag and import it as business is driving the changes needed